plugins/confluence-plugin/README.md

```diff
@@ -6,25 +6,42 @@ This plugin makes it possible to view Confluence assets alongside related entiti
 
 This is a [Cortex](https://www.cortex.io/) plugin. To see how to run the plugin inside of Cortex, see [our docs](https://docs.cortex.io/docs/plugins).
 
-### Prerequisites
+## Setup
 
-The URL of your Confluence instance should be configured in `src/components/PageContent.tsx`. To do so, change the following line to match the URL of your server:
+### Find your Confluence instance URL
 
-```
-const baseConfluenceUrl = "https://confluence-server.atlassian.net";
-```
+Your Confluence instance URL should look like `https://something.atlassian.net`. We will use this URL in the plugin's proxy and in the plugin's configuration entity.
+
+### Set your Confluence credentials
+
+If you are using username and password authentication, type them in to a text editor with a colon (`:`) between them. If you are using SSO, use an API token in place of the password. To create an API token in Confluence by clicking on Security > Create and manage API tokens > Create API token. Once you have your text in your text editor like `myusername@example.com:MySecretPassWordOrToken`, you need to base64 encode it. You can do this using an online tool like [this](https://www.base64encode.org). You will add the base64 encoded value to Cortex as a secret. Copy the base64 encoded value. In Cortex, click on Settings > Secrets > Add secret. In Secret name, type `confluence_secret`, paste the base64 encoded secret into Secret value, and click on Create secret.
```

Comment:
Nitpick, typo, should be: "To create an API tolein in Confluence, click on Security > Create"